src/styles/layout.scss

```diff
@@ -497,3 +499,9 @@ details {
     padding-right: 0;
   }
 }
+
+#layout-container {
```

Comment:
Why are you using an ID over a class?

Reply:
Is there a preference? I thought ID would be more appropriate because there would only be 1 layout component.

Reply:
The preference is classes.

Reply:
Updated to class